### PR TITLE
Made updates to current issues in Linked-List file

### PR DESCRIPTION
**Pull request by Carloscode5**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Carloscode5** [comment]
Made requested changes to markdown file.

**Comment by dakshdeepHERE** [comment]
@Carloscode5 the `tags.md` file is already there in the documentation folder. https://github.com/Codecademy/docs/tree/main/documentation See here. You have to add the tags there. I will go through the PR again. Why you created a new PR instead of not working on the older one?

File: content/c/concepts/linked-list/linked-list.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+    list = push(list, "World ");
+    list = push(list, "Hello ");
+-    //print out the elements
+    Node* crnt = list;
+    while(crnt != NULL){
+        printf("%s", crnt->data);
+        crnt = crnt->next;
+    }
+    return 0;
+}
```
Comment (dakshdeepHERE) [inline content/c/concepts/linked-list/linked-list.md:49]:
```suggestion
struct Node* push(struct Node* head, void* data) {
  struct Node* new_node = (struct Node*)malloc(sizeof(struct Node));
  new_node->data = data;
  new_node->next = head;
  return new_node;
}
int main(){
  Node* list = NULL;
-    //remember to set the list after push
  list = push(list, "!");
  list = push(list, "World ");
  list = push(list, "Hello ");
-    //print out the elements
  Node* crnt = list;
  while(crnt != NULL){
      printf("%s", crnt->data);
      crnt = crnt->next;
  }
  return 0;
}
```

File: content/c/concepts/linked-list/linked-list.md [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+```
+
+## Example
```
Comment (dakshdeepHERE) [inline content/c/concepts/linked-list/linked-list.md:52]:
```suggestion
## Example

```

File: content/c/concepts/linked-list/linked-list.md [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+      printf("%d->",temp->data);
+      temp = temp->next;
+  }
+  printf("NULL");
+
+  return 0;
+}
+```
+
+### Singly linked lists
```
Comment (dakshdeepHERE) [inline content/c/concepts/linked-list/linked-list.md:101]:
```suggestion
### Singly linked lists

```

File: content/c/concepts/linked-list/linked-list.md [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+  }
+
+  while (head != NULL) {
+    printf("%d\n", head->data);
+    head = head->next;
+  }
+}
+```
+
+### Doubly linked lists
```
Comment (dakshdeepHERE) [inline content/c/concepts/linked-list/linked-list.md:138]:
```suggestion
### Doubly linked lists

```

File: content/c/concepts/linked-list/linked-list.md [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+  insert_at_beginning(&head, 1);
+  insert_at_beginning(&head, 2);
+  insert_at_end(&head, 3);
+  insert_at_end(&head, 4);
+  print_list(head);
+  return 0;
+}
+```
+
+### Circular linked lists
```
Comment (dakshdeepHERE) [inline content/c/concepts/linked-list/linked-list.md:191]:
```suggestion
### Circular linked lists

```

File: content/c/concepts/linked-list/linked-list.md [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+int main() {
+  insert(1);
+  insert(2);
+  insert(3);
+  print();
+  return 0;
+}
+```
+
+### Circular doubly linked lists
```
Comment (dakshdeepHERE) [inline content/c/concepts/linked-list/linked-list.md:241]:
```suggestion
### Circular doubly linked lists

```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Work on these comments please. Then will go through the entry again.

**Comment by SSwiniarski** [comment]
> @Carloscode5 the `tags.md` file is already there in the documentation folder. https://github.com/Codecademy/docs/tree/main/documentation See here. You have to add the tags there. I will go through the PR again. Why you created a new PR instead of not working on the older one?

@dakshdeepHERE, just an FYI, I closed the earlier duplicate PR.

**Comment by Carloscode5** [comment]
Thank you Dar, working on this now!Thank you so much. I seriously went through everything for my first go at contributing thank you.My next one will be bullet proof!Working on your requests now.Best,Carlos ValenzuelaOn Nov 28, 2023, at 9:30 AM, SSwiniarski ***@***.***> wrote:﻿

@Carloscode5 the tags.md file is already there in the documentation folder. https://github.com/Codecademy/docs/tree/main/documentation See here. You have to add the tags there. I will go through the PR again. Why you created a new PR instead of not working on the older one?

@dakshdeepHERE, just an FYI, I closed the earlier duplicate PR.

**Review by Carloscode5 — COMMENTED** [review]
Addressed all corrections requested, please advise.

**Comment by Carloscode5** [comment]
> Work on these comments please. Then will go through the entry again.

Addressed all requests, please let me know if I missed anything else.

File: content/c/concepts/linked-list/linked-list.md [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+}
+```
+
+## Example
+
+The following C code showcases the creation and traversal of a linked list, which is comprised of a series of nodes with integer values of 10, 20 and 30. The nodes are connected continuously from head to tail displaying the elements of the linked list.
+
+```c
+#include
+#include
```
Comment (dakshdeepHERE) [inline content/c/concepts/linked-list/linked-list.md:58]: What are these headers? I don't think these are completed please fix this.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have one last comment for you to work on @Carloscode5

File: content/c/concepts/linked-list/linked-list.md (hunk 7 above)
Comment (Carloscode5) [reply content/c/concepts/linked-list/linked-list.md]: Hi Dak, thank you for reviewing my latest updates. I went ahead and fixed headers. Please review and let me know if anything else needs review. I went ahead and added along w/ commited and pushed to main branch.

Please advise :)

**Review by dakshdeepHERE — APPROVED** [review]
This looks good to move for second review now. Thank you @Carloscode5 for working on the commits. 🚀

**Comment by Carloscode5** [comment]
Thank you Dak for your patience and dedication!Wonderful; what’s next for me as far as steps go?Best,Carlos ValenzuelaOn Dec 15, 2023, at 9:59 AM, Daksha Deep ***@***.***> wrote:﻿
@dakshdeepHERE approved this pull request.

This looks good to move for second review now. Thank you @Carloscode5 for working on the commits. 🚀

File: content/c/concepts/linked-list/linked-list.md [hunk 8]
```diff
@@ -0,0 +1,329 @@
+---
+Title: 'Linked List'
+Description: 'A data structure that contains a collection of nodes that together with the help of pointers, display a sequence.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Singly linked lists'
```
Comment (SSwiniarski) [inline content/c/concepts/linked-list/linked-list.md:8]: These are not legit tags. Tags must come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).

File: content/c/concepts/linked-list/linked-list.md [hunk 9]
```diff
@@ -0,0 +1,329 @@
...
+  - 'Doubly linked lists'  
+  - 'Circular linked lists'
+  - 'Circular doubly linked lists'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+A **`Linked List`** is a data structure that contains a collection of `nodes` that together with the help of pointers, display a sequence. There are several list types, we will cover four of the most common with examples in the [C language](https://www.codecademy.com/resources/docs/c).
```
Comment (SSwiniarski) [inline content/c/concepts/linked-list/linked-list.md:17]:
```suggestion
A **`Linked List`** is a data structure that contains a collection of `nodes` that together with the help of pointers, display a sequence. There are several list types, four of the most common are discussed below.
```
Avoid first and second person pronouns in docs, also this is a C entry, there's no need to reiterate the examples are in C.

File: content/c/concepts/linked-list/linked-list.md [hunk 10]
```diff
@@ -0,0 +1,329 @@
...
+
+## Syntax
```
Comment (SSwiniarski) [inline content/c/concepts/linked-list/linked-list.md:19]: The Syntax section seems to be superfluous given the below examples. Usually a Syntax section really only applies to an illustration of using a built-in function or method, where this is more an illustration of linked lists in C.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Carloscode5 I did a second review and had a few comments.

File: content/c/concepts/linked-list/linked-list.md (hunk 8 above)
Comment (Carloscode5) [reply content/c/concepts/linked-list/linked-list.md:8]: Hi @SSwiniarski thank you for your review. Are you referring to everything on lines 5-8? If so, what would you recommend or would you have any suggestions on what I should replace with?

File: content/c/concepts/linked-list/linked-list.md (hunk 9 above)
Comment (Carloscode5) [reply content/c/concepts/linked-list/linked-list.md:17]: This change has been update and pushed for review.

File: content/c/concepts/linked-list/linked-list.md (hunk 10 above)
Comment (Carloscode5) [reply content/c/concepts/linked-list/linked-list.md:19]: @SSwiniarski For this one I'm not quite sure how to turn this into a simple syntax without removing how it works. What do you recommend or have as a rough example to replace?

**Comment by ishg-153** [comment]
@Carloscode5 are you done making the changes suggested in the second review?

**Comment by Carloscode5** [comment]
Hi there,Just have to make final changes then will push for review.Best,Carlos ValenzuelaOn May 15, 2024, at 12:23 AM, Ishwari Garge ***@***.***> wrote:﻿
@Carloscode5 are you done making the changes suggested in the second review?

**Comment by ishg-153** [comment]
@Carloscode5 It's been 3 weeks. I will need to close this PR if there is no revert from you in the next 2 days.

**Comment by ishg-153** [comment]
Closing this PR due to lack of activity.

**Pull request closed without merging**